examples/MQ/pixelDetector/src/FairOnlineSink.cxx

```diff
@@ -19,15 +19,9 @@
 
 #include <fairlogger/Logger.h>
 
-FairOnlineSink::FairOnlineSink()
-    : FairSink()
-    , fMQRunDevice(nullptr)
-{}
+FairOnlineSink::FairOnlineSink() = default;
```

Comment:
I think the `= default` needs to go to the header file?

Reply:
I think, both is allowed. But yes, putting it in the header allows more optimizations. So that's a good thing.

Done.